runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -106,6 +106,16 @@ void deactivateDevice()
 
 extern "C" {
 
+void pyregistry(void *p, uintptr_t *aligned, int64_t i)
+{
+    void* handle = dlopen("registry.cpython-310-x86_64-linux-gnu.so", RTLD_LAZY | RTLD_NODELETE);
```

Comment:
Update this to be OS specific. (But can we avoid python version here? Maybe a simple regex could work?)